packages/web-components/fast-foundation/src/accordion-item/accordion-item.ts

```diff
@@ -38,7 +34,7 @@ export type AccordionItemOptions = FoundationElementDefinition &
  *
  * @public
  */
-export class AccordionItem extends FoundationElement {
+export class FoundationAccordionItem extends FoundationElement {
```

Comment:
Is there a *non* foundation Accordion Item that we'll be exporting as well? If not, we don't have our own implementation of this anymore, so this could technically be `FASTAccordionItem`. I think it could be interesting to have an implementation that isn't tightly coupled to Foundation, though I know a lot of what may have been considered "bloat" there by some is moving out. Consider a scenario where I want to create an Accordion but only want to extend of FASTElement...in that case, we could have `FASTAccordionItem` and `FoundationAccordionItem`. If we want to keep it to just one, I might suggest using the FAST prefix here since it represents THE component implementation we have.

Reply:
I can easily change this to use `FAST` as a prefix. There won't be much in `FoundationElement` after the refactor. Basically just the `style` and `template` properties to override those on a per instance. We don't actually need those though, so we could remove `FoundationElement` altogether if we want. Thoughts?